DTS-STN/dts-rmp
# New103

# Description

- call API to fetch contacts and populate dropdown and display contact & enagement info of selected contact
- updated with main branch
- added some stylings on the engagement form
 
Note: This is the updated version of my previous pr. Fixed some bugs that Beniot pointed out earlier.

## Test Instructions

- navigate to the add contacts & engagement page, select engagement
- play with the dropdown to see if contacts displayed properly
- engagement info should not be displayed if the contact selected doen't have any engagements 

## Reviewers
@bcloutier7 @P0NDER0SA @will0684 @andr0272 @MarcoGoC @nghe0001 @shewood 

## Checklist
